packages/webui/src/components/Button.tsx

```diff
@@ -3,12 +3,14 @@ import { MouseEvent, Ref, RefObject, forwardRef, useRef } from 'react';
 import { tw } from '@/utils';
 
 export type ButtonProps = React.ButtonHTMLAttributes<HTMLButtonElement> & {
-  size?: 'small' | 'standard' | 'large';
-  border?: 'standard' | 'ghost';
+  border?: 'standard' | 'none';
+  Icon?: React.FC<any>;
```

Comment:
do we need to pass `any` here or could we do something like `Record<string, unknown>`?